geometry_msgs/Point start_point
geometry_msgs/Point goal_point
---
bool success